hack/e2e/cluster/tools/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func main() {
+
+	encodedStr := "YXBpVmVyc2lvbjogdjEKZGF0YToKICBsaWNlbnNlOiBleUpzYVdObGJuTmxJanA3SW5CaGVXeHZZV1FpT25zaVlXUnRhVzVmYzJWaGRITWlPaUkxSWl3aVkzVnpkRzl0WlhJaU9pSmhkWFJ2YldGMGFXOXVJaXdpWkdGMFlYQnNZVzVsY3lJNklqRXdNQ0lzSW14cFkyVnVjMlZmWTNKbFlYUnBiMjVmWkdGMFpTSTZJakl3TWpFdE1EY3RNallpTENKc2FXTmxibk5sWDJWNGNHbHlZWFJwYjI1ZlpHRjBaU0k2SWpJd01qRXRNVEF0TXpFaUxDSnNhV05sYm5ObFgydGxlU0k2SWtGVFJFRlRSRUZUUkVGVFJFRlRSRUZUUkVGVFJFRlRSRUZUUkY5aE1WWkJVMEZUUkNJc0luQnliMlIxWTNSZmMzVmljMk55YVhCMGFXOXVJam9pUzI5dWJtVmpkQ0JGYm5SbGNuQnlhWE5sSWl3aWMzVndjRzl5ZEY5d2JHRnVJam9pVG05dVpTSjlMQ0p6YVdkdVlYUjFjbVVpT2lKaFptTTBZVEl6TmpneE9EWXhORGhtT0RBNE9UTXhaREEwWkRFM05qQTRZbVExT0RRek1qTXlPVGMyWlRJM09XVm1PR0l6TUdSbVpXUTFZV1ZrTmpNeU1XVmtOMlZpWWpBM1pHTmtPRGRoWWprNU5tVTNaalEzTVdReE56Tm1ObUV6WkdGa1l6STVNbVl5TjJOaVpHSXpaRE5rWm1FMk9ERmtNelprWVdaak15SXNJblpsY25OcGIyNGlPaUl4SW4xOUNnPT0Ka2luZDogU2VjcmV0Cm1ldGFkYXRhOgogIG5hbWU6IGtvbmctZW50ZXJwcmlzZS1saWNlbnNlCnR5cGU6IE9wYXF1ZQoK"
```

Comment:
This secret should not have been committed to git: we need to invalidate this license and and then load this value as an environment secret (see the `release.yaml` CI) instead.

Reply:
This is base64 encoded.